beacon_node/eth2-libp2p/src/rpc/protocol.rs

```diff
@@ -168,12 +190,18 @@ impl RPCRequest {
     pub fn supported_protocols(&self) -> Vec<ProtocolId> {
         match self {
             // add more protocols when versions/encodings are supported
+            RPCRequest::Hello(_) => vec![ProtocolId::new("hello", "1", "ssz_snappy")],
```

Comment:
This function is used on line 184 above. Basically for each request we receive we need to tell libp2p which protocols we support. Here these are stored as a list, and we should add protocols to the list for each request. 

Currently, this match would only ever return the `ssz_snappy` protocol id and cannot return the `ssz` anymore.